### PR TITLE
add workflow to build and deploy docs

### LINKED ISSUE #825: Create pipelines to automatically build documentation
_Originally posted by @vigneshhari in https://github.com/coronasafe/care/issues/763#issuecomment-1147457139_

### PR DESCRIPTION
**Pull request by sainAk**
closes #825

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: Build Docs
+
+on:
+  push:
```
Comment (vigneshhari) [inline .github/workflows/docs.yml:4]: we only need to build/deploy on the master branch

**Review by vigneshhari — CHANGES_REQUESTED** [review]

**Review by Marmik2003 — APPROVED** [review]
Looks good, i think it is ok if we don't remove sphinx from local requirements as some developers might also want to change the docs after few major changes!

**Comment by sainAk** [comment]
> Looks good, i think it is ok if we don't remove sphinx from local requirements as some developers might also want to change the docs after few major changes!

I've refrenced the docs.txt requirements in local.txt so sphinx will also get installed in local environments

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+      - name: Upload sphinx documentation
+        uses: actions/upload-artifact@v3
+        with:
+          name: sphinx-docs
+          path: ./docs/_build
+          retention-days: 30
+
+  deploy-docs:
+    if: github.ref == 'refs/heads/master'
```
Comment (vigneshhari) [inline .github/workflows/docs.yml:43]: Do we need this, this job only runs on pushes to master right?

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**